src/maintainer/knowledge_base.rst

```diff
@@ -554,45 +554,70 @@ Special packages
+host of the recipe,
+
+.. code-block:: yaml
+    requirements:
+      host:
+        - libblas
+        - libcblas
+        - liblapack
+        - liblapacke
+
+Note that you should specify only the libraries the package needs. (i.e. if the package
+doesn't need LAPACK, remove liblapack and liblapacke)
+
+At recipe build time, above requirements would download the NETLIB's reference
+implementations and build your recipe against those. At runtime by default the following packages
+will be used,
+
+.. code-block:: yaml
+    - openblas   # [not win]
+    - mkl        # [win]
 
-  The information regarding the blas package may now be outdated and needs checking.
+You can also switch your BLAS implementation by doing,
 
-BLAS metapackage
-................
+.. code-block:: bash
+    conda install "libblas=*=*mkl"
+    conda install "libblas=*=*openblas"
+    conda install "libblas=*=*blis"
+    conda install "libblas=*=*netlib"
+
+Following legacy commands are also supported for now,
+
+.. code-block:: bash
+    conda install "blas=*=mkl"
+    conda install "blas=*=openblas"
+    conda install "blas=*=blis"
+    conda install "blas=*=netlib"
 
-   -  Version will have two values X.Y
+If a package needs a specific implementation's internal API for more control you can have,
 
-      -  X represents changes to the metapackage.
-      -  Y represents priority of BLAS (if we change priorities BLASes X
-         must be bumped, if we want to prioritize something new over
-         OpenBLAS we do not need to change X)
+.. code-block:: yaml
+    requirements:
+      host:
+        - {{ blas_impl }}
+      run:
+        - libblas * *{{ blas_impl }}
+        - {{ blas_impl }}
 
-         -  1 is OpenBLAS
-         -  0 is None (no BLAS whatsoever)
+This would give you a matrix builds for different blas implementations. If you only want to support
+a specific blas implementation,
 
-   -  needs to have version stay the same across all variants.
-   -  build number cannot be touched (it won't be in the string anyways)
-   -  no pinning of BLAS inside the metapackages (dependencies can pin
-      this)
-   -  to preserve a BLAS in an environment it is recommend to add
-      ``pinned`` to ``conda-meta`` and specify down to the build string
-      what is the expected BLAS
-   -  To install a specific variant, ``conda install blas=1.0=none`` /
-      ``conda install blas=1.0=openblas``. It is hoped the syntax will be
-      improved in conda.
+.. code-block:: yaml
+    requirements:
+      host:
+        - openblas
+      run:
+        - libblas * *openblas
+        - openblas
 
-      -  In the future, with some fixes to ``conda`` will allow for syntax
-         like ``conda install blas=*=openblas``. We should keep an eye on
-         this. (maybe even ``conda install blas:openblas``)
 
-   -  There will be two variants initially:
+Note: `blas_*` features should not be used anymore.
```

Comment:
This could also be done in a ``.. note::`` directive for some additional visual sugar :-)